app/src/main/java/com/rockthevote/grommet/ui/MainActivityViewModel.kt

```diff
@@ -113,6 +113,17 @@ class MainActivityViewModel(
         }
     }
 
+    fun updateSessionStatus(sessionStatus: SessionStatus){
+        viewModelScope.launch(dispatchers.io) {
+            /*
+            TODO - how to implement this method?
```

Comment:
Just waned to call out this comment

Reply:
Echoing what's in the Slack channel. SharedPreferences seems like the best way to go on this. We can just store the status as a single entry which can be pretty easily observed throughout the app as necessary.